ITensorGPU/src/tensor/cudense.jl

```diff
@@ -567,7 +567,10 @@ function Base.permute!(B::CuDense, Bis::IndexSet, A::CuDense, Ais::IndexSet)
     reshapeBdata,
     Vector{Char}(ctbinds),
   )
-  return Tensor(Bis, Dense(vec(reshapeBdata)))
+  @show Tuple(Bis)
```

Comment:
Just noting these printing statements so you remove them before the PR is finalized, and the one below showing typeof(d)

Reply:
Thanks for pointing this out! I removed this code